packages/_ssl/meta.yaml

```diff
@@ -1,5 +1,5 @@
 package:
-  name: ssl
+  name: _ssl
```

Comment:
This change will cause a bit of a regression with `loadPackageFromImports`. Currently `import ssl` in `console.html` works, but if you change the name to `_ssl` then it will fail.

Reply:
Yes, good point. I misread it as being changed the other way around.

Reply:
`import ssl` will still work in the console since `loadPackageFromImports`  uses `test/imports` key to determine the top-level import name and I didn't change it. Though we probably need to change its behavior later (https://github.com/pyodide/pyodide/pull/2581#discussion_r878958962)